<ui version="4.0" >
 <class>ClsQHarbor</class>
 <widget class="QDialog" name="ClsQHarbor" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>606</width>
    <height>279</height>
   </rect>
  </property>
  <property name="acceptDrops" >
   <bool>true</bool>
  </property>
  <property name="windowTitle" >
   <string>iqr Harbor</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="spacing" >
    <number>0</number>
   </property>
   <property name="margin" >
    <number>0</number>
   </property>
   <item>
    <widget class="QTableWidget" name="tableWidget" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
       <horstretch>0</horstretch>
       <verstretch>10</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QPushButton" name="pbDeleteRow" >
       <property name="text" >
        <string>&amp;Delete Row(s)</string>
       </property>
       <property name="shortcut" >
        <string>Alt+D</string>
       </property>
       <property name="default" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbRefresh" >
       <property name="text" >
        <string>Refresh</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>175</width>
         <height>21</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbLoadConfig" >
       <property name="text" >
        <string>&amp;Load Config</string>
       </property>
       <property name="shortcut" >
        <string>Alt+L</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbSaveConfig" >
       <property name="text" >
        <string>&amp;Save Config</string>
       </property>
       <property name="shortcut" >
        <string>Alt+S</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QPushButton" name="pbLoadParamSet" >
       <property name="text" >
        <string>Load Parameters</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbSaveParamSet" >
       <property name="text" >
        <string>Save Parameters</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>330</width>
         <height>21</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <resources/>
 <connections/>
</ui>
